.circleci/config.yml

```diff
@@ -310,6 +314,40 @@ jobs:
           paths:
             - /home/circleci/cache
 
+  # Auth to GCR, tag and push image
+  deploy-to-gar:
+    executor: gcp-gcr/default
+    parameters:
+      image:
+        type: string
+        default: us-docker.pkg.dev/my-gcp-project-id/my-gar-repo/my-image
```

Comment:
This value feels very "template-y" is it supposed to be or should it have more "real" values?

Reply:
gah! thank you